README.md

```diff
@@ -32,9 +33,20 @@ cd sourcery-api/
 // install dependencies
 bundle
 
+// create db and migrate
+rake db:{create,migrate}
```

Comment:
@tmikeschu I accept that this is a nitpick, but to offer an alternative to the above command, what about: `rake db:{create,schema:load}` ?  This loads the schema based on what is in schema.rb as opposed to running all of the migrations.